custom-connectors/CopilotForFinanceCommunications/apiProperties.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please refer the below schema link and update this file. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json

Reply:
updated